rasa_nlu/data_router.py

```diff
@@ -210,7 +235,9 @@ def start_train_process(self, data, config_values):
             _config[key] = val
         _config["data"] = f.name
         train_config = RasaNLUConfig(cmdline_args=_config)
-        process = multiprocessing.Process(target=do_train, args=(train_config, self.component_builder))
-        self._add_train_proc(process)
```

Comment:
We don't necessarily need to directly track the processes as you said. I totally agree with you there, as long as we keep track of trainings at some location that is totally fine.